PlotNeuralNetPy/pyexamples/__init__.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from .test_simple import main, arch
```

Comment:
These imports conflict.  No need to import the individual example namespaces into the top `__init__`  namespace.

Reply:
Thanks! Didn't catch that!